hashes/src/sha256.rs

```diff
@@ -817,7 +817,8 @@ impl HashEngine {
 
 #[cfg(test)]
 mod tests {
-    use crate::{sha256, Hash, HashEngine};
+    use crate::{sha256, Hash as _, HashEngine};
```

Comment:
Honestly, I hate using `crate::foo` where `foo == super`. But I'll give it a pass this time since I don't think `use super` will add `sha256` into the scope.